source/includes/fact-export-selection.rst

```diff
@@ -0,0 +1,6 @@
+a. When you highlighted your code, a light bulb icon appeared. Click
```

Comment:
This include appears at the top of step 2 in both procedures.

Reply:
Staged here:

- https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker-xlarge/DOCSP-19772/export-to-language/#export-your-selection
- https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker-xlarge/DOCSP-19772/export-to-language/#export-your-selection-1

Reply:
Done

Reply:
(I don't see a way to resolve the comment in this side-by-side view. I'll check the conversation view afterwards.